docs/proposals/002-api-proposal/glossary.md

```diff
@@ -0,0 +1,94 @@
+# Glossary
+
+This is a glossary that attempts to more thoroughly emplain terms used within the api proposal, in an effort to give context to API decisions.
+
+<!-- toc -->
+- [API Terms](#api)
+    - [BackendPool](#backendpool)
+    - [UseCase](#UseCase)
+- [Capacity Constrained Routing](#capacity-constrained-routing)
+    -   [Priority](#priority)
+    -   [Fairness](#fairness)
+- [General Routing](#general-routing)
+    -   [Latency Based Routing](#latency-based-routing)
+    -   [Lora Affinity](#lora-affinity)
+
+
+<!-- /toc -->
+
+## API
+This is a very brief description of terms used to describe API objects, included for completeness.
+
+### BackendPool
+A grouping of model servers that serve the same set of fine-tunes (LoRA as a primary example). 
+
+Shortened to: `BEP`
```

Comment:
Can we use `BP` instead?  We're using "backend" as a single word, this initialism is somewhat inconsistent.  I recognize we call kube `EndpointSlices` `eps` in short CLI form, but that's mostly because we called `Endpoints` `ep` to have a two letter version.

Reply:
Yes, apologies. Blame mountain biking. It always gets shortened to MTB despite that making no sense. So I picked up the bad habit.